docs/audit-log.rst

```diff
@@ -0,0 +1,49 @@
+Audit Log Notifications
```

Comment:
I'm hesitant to use the word notifications as that is a different CBC concept and is not used in conjunction with Audit logs in the User Guide. Can we try to scrub all references to notification in the audit log guide/docstrings

Reply:
The word "notifications" is used on Dev Network for this API, too: https://developer.carbonblack.com/reference/carbon-black-cloud/cb-defense/latest/rest-api/#audit-log-events

Reply:
But I removed the word "notifications" entirely.

Reply:
That is our legacy API documentation and the future is to move away from integrationServices so I'd like to move towards the same terminology used in the User Guide which does not use notifications